### PR TITLE
CNV-37647: Added table comparing bridge CNI and OVN-K localnet

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.17+

Issue: [CNV-37647](https://issues.redhat.com//browse/CNV-37647)

Link to docs preview: https://79192--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-networking-overview.html#comparing-bridge-localnet

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/vm_networking/virt-networking-overview.adoc [hunk 1]
```diff
@@ -85,6 +85,41 @@ endif::openshift-rosa,openshift-dedicated[]
...
+|No
+|Yes
+
+|Managed IP pools
+|No
+|Yes
+
+|MAC spoof filtering
+|Yes
+|No
```
Comment (maiqueb) [inline virt/vm_networking/virt-networking-overview.adoc:118]: wait, this is not right. We do support MAC spoof filtering on localnet.

**Review by maiqueb — APPROVED** [review]
Thank you.

**Comment by sjhala-ccs** [comment]
@yossisegev Can you please review?

**Comment by sjhala-ccs** [comment]
@maiqueb @yossisegev I have created separate [4.15](https://github.com/openshift/openshift-docs/pull/79378) and [4.16-specific](https://github.com/openshift/openshift-docs/pull/79377) PRs where the support for Managed IP pools is marked as "No" on OVN-K localnet. PTAL. Thank you.

File: virt/vm_networking/virt-networking-overview.adoc [hunk 2]
```diff
@@ -85,6 +85,43 @@ endif::openshift-rosa,openshift-dedicated[]
 
 . xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#attaching-vm-to-ovn-secondary-nw[Connect the VM to the OVN-Kubernetes secondary network] by adding the network details to the VM specification.
 
+// Hiding from ROSA/OSD as Linux Bridge is not supported
+ifndef::openshift-rosa,openshift-dedicated[]
+[id="comparing-bridge-localnet"]
+=== Comparing Linux bridge CNI and OVN-Kubernetes localnet topology
+
+The following table provides a comparison of features available when using the Linux bridge CNI versus the localnet topology for an OVN-Kubernetes plug-in:
```
Comment (dfitzmau) [inline virt/vm_networking/virt-networking-overview.adoc:93]:
```suggestion
The following table provides a comparison of features available when using the Linux bridge CNI versus the localnet topology for an OVN-Kubernetes plugin:
```
Consistent with named term in core docs: https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html

File: virt/vm_networking/virt-networking-overview.adoc [hunk 3]
```diff
@@ -85,6 +85,43 @@ endif::openshift-rosa,openshift-dedicated[]
...
+
+.Linux bridge CNI compared to an OVN-Kubernetes localnet topology
+[cols="1,1,1",options="header"]
+|===
+|Feature
+|Available on Linux bridge CNI
+|Available on OVN-Kubernetes localnet
+
+|Layer 2 access to the underlay native network	
+|Only on secondary Network Interface Cards (NICs)
```
Comment (dfitzmau) [inline virt/vm_networking/virt-networking-overview.adoc:103]: See https://redhat-documentation.github.io/supplementary-style-guide/#network-interface-card.

**Review by dfitzmau — COMMENTED** [review]
Hi @sjhala-ccs . Nice work on the table. Added comments inline.

/remove-label peer-review-in-progress

/label peer-review-done

**Comment by Anatw** [comment]
Nice work, Shikha :)

/lgtm

**Comment by maiqueb** [comment]
Wait ... so we somehow managed to merge the 4.15 and 4.16 docs before the 4.17 (master) ? 

How could this happen ? It really doesn't feel normal. 

Still, I don't think it will cause any issues ... Still ... I'm puzzled.

**Comment by sjhala-ccs** [comment]
> Wait ... so we somehow managed to merge the 4.15 and 4.16 docs before the 4.17 (master) ?
> 
> How could this happen ? It really doesn't feel normal.
> 
> Still, I don't think it will cause any issues ... Still ... I'm puzzled

The 4.15 and 4.16-specific tables were in separate PRs. And I had to make an update to this one because of merge conflicts caused by a different PR.

**Comment by maiqueb** [comment]
> > Wait ... so we somehow managed to merge the 4.15 and 4.16 docs before the 4.17 (master) ?
> > How could this happen ? It really doesn't feel normal.
> > Still, I don't think it will cause any issues ... Still ... I'm puzzled
> 
> The 4.15 and 4.16 tables were in separate branch-specific PRs. And I had to make an update to this one because of merge conflicts caused by a different PR.

I see. 

The process we follow in the openshift components (code) is to merge first to the `main` branch, and only then issue the backports to the release branches (by issueing the cherry pick command, which the openshift bot knows how to interpret).

This prevents this situation from happening, since the bot will not open the cherry pick / backport PRs until the original PR is merged. 

Food for thought for next time :) 

Thanks for adding these tables. 

/lgtm

**Pull request merged**